lib/manager/kubernetes/index.ts

```diff
@@ -1,10 +1,11 @@
 import { LANGUAGE_DOCKER } from '../../constants/languages';
 
 export { extractPackageFile } from './extract';
-export { updateDependency } from './update';
 
 export const language = LANGUAGE_DOCKER;
 
+export const autoReplace = true;
```

Comment:
Flip default value and assume false if update function exists?

Reply:
Yes, I'm going to do that eventually but will leave it as-is for now.